driver/collector/postgres_collector.py

```diff
@@ -389,7 +392,7 @@ def __init__(
                 "FROM pg_stat_statements;"
             )
 
-    def _cmd(self, sql: str):  # type: ignore
+    def _cmd(self, sql: str, logical_db: str = None):  # type: ignore
```

Comment:
the type here should be an `Optional[str]` I think instead of str. I know we have the type: ignore comment here but that is more accurate.

Reply:
On another note... i feel like it is cleaner to have this require a logical_db and not make it optional. Callers should be able to pass in which logical_db they'd like (whether that is `self.main_logical_db` or something else.

Reply:
Made in non-optional.